### ISSUE TITLE
Coming and leaving

### ISSUE THREAD

**Issue by diyar1122** [changeweb/Unifiedtransform#370]
How do implement users coming and leaving on Unified Transform?

**Comment by yungifez** [comment]
I dont think that feature exists

**Comment by changeweb** [comment]
> How do implement users coming and leaving on Unified Transform?

Do you mean login and logout?

**Comment by yungifez** [comment]
I dont think that feature existsi think he means graduating

**Comment by diyar1122** [comment]
No, I mean how can a student or teacher come or go from the platform.

**Comment by changeweb** [comment]
Both students and teachers are registered. As for students, suppose you registered and promoted a student to a section in a class for a session. If the student leaves the school, his/her record will be only for the sessions he/she was registered and promoted to.

When a student graduates, the admin no longer promotes him/her to any new session.
